src/swdmt/redistricting/DistrictTest.java

```diff
@@ -1,8 +1,9 @@
 package swdmt.redistricting;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.assertFalse;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
 import org.junit.Test;
+//import org.junit.jupiter.api.Test;
```

Comment:
Why is this reference commented out.  If it is not necessary, it should be removed.

Same for other commented out imports in other files

Reply:
I was trying to change org.junit.Test to org.junit.jupiter.api.Test but when I did I would get a message that said
_"Testcase: initializationError took 0.01 sec
         Caused an ERROR
     No runnable methods
     java.lang.Exception: No runnable methods"_

So I had it commented out until I could pinpoint the issue.

Reply:
That's fine, since we know this to be an incomplete PR used for practice.
This sub-conversation would remain open until the particular concern was addressed.